tests/UIButtons.js

```diff
@@ -0,0 +1,173 @@
+import Menu from "../page-objects/components/Menu";
+import HomePage from "../page-objects/pages/Homepage";
+
+describe("Checking the appearance of buttons", () => {
+  beforeEach(async () => {
+    HomePage.openHomePage();
+    Menu.waitForExistHeader();
+    Menu.clickOnApproveButton();
+    Menu.clickOnApproveButtonFirefox();
+    await browser.pause(5000);
+  });
+
+  it("Checking the appearance of the menu buttons", async () => {
+    const buttons = [
+      Menu.cuteButton,
+      Menu.funnyButton,
+      Menu.goodButton,
+      Menu.omgButton,
+      Menu.moreButton,
+    ];
+
+    for (let i = 0; i < buttons.length; i++) {
+      const elem = buttons[i];
+
+      const fontSize = await elem.getCSSProperty("font-size");
+      console.log(fontSize);
+      const valueFontSize = fontSize.value;
+      await expect(valueFontSize).toEqual("16px");
+
+      const minWidth = await elem.getCSSProperty("min-width");
+      console.log(minWidth);
+      const valueMinWidth = minWidth.value;
+      await expect(valueMinWidth).toEqual("120px");
+
+      const borderLeft = await elem.getCSSProperty("border-left-width");
+      console.log(borderLeft);
+      const valueBorderLeft = await borderLeft.value.slice(0, -2);
+      await expect(Math.round(valueBorderLeft)).toEqual(2);
+
+      const borderTop = await elem.getCSSProperty("border-top-width");
+      console.log(borderTop);
+      const valueBorderTop = await borderTop.value.slice(0, -2);
+      await expect(Math.round(valueBorderTop)).toEqual(5);
+
+      const color = await elem.getCSSProperty("color");
+      console.log(color);
+      const valueColor = color.parsed.hex;
+      await expect(valueColor).toEqual("#898989");
+
+      const alignItems = await elem.getCSSProperty("align-items");
+      console.log(alignItems);
+      const valueAlignItems = alignItems.value;
+      await expect(valueAlignItems).toEqual("center");
+    }
+  });
+
+  it("Checking the appearance of the submenu buttons", async () => {
+    const submenuButtons = [
+      await Menu.aboutButton,
+      Menu.privacyPolicyButton,
+      Menu.dmcaButton,
+      Menu.advertisingButton,
+    ];
+
+    await (await Menu.moreButton).waitForDisplayed();
+    await (await Menu.moreButton).click();
+    await browser.pause(5000);
```

Comment:
This pause looks to be for too many seconds. Maybe you could wait for the submenu to be displayed.